src/test/java/com/snowflake/kafka/connector/internal/streaming/SnowflakeSinkServiceV2IT.java

```diff
@@ -63,31 +62,34 @@ public class SnowflakeSinkServiceV2IT {
   private TopicPartition topicPartition = new TopicPartition(topic, partition);
 
   // use OAuth as authenticator or not
-  private boolean useOAuth;
+  private boolean useSingleBuffer = true;
```

Comment:
We pass `useSingleBuffer` to each test as an argument. Does need to be a class field too?